python/tests/_utils/test_comparision.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import scipp as sc
+import scipp._utils as su
```

Comment:
Why not provide `is_near` in the scipp module? Is this so special that it should stay in a protected module?

Reply:
I imagine developers only want to use this for testing, but maybe I'm wrong about this?

Reply:
I like the two-stage approach: First introduce it as a private helper, and once it is consolidated and we know what it is, we can consider making it public. `isclose` is different, since it has a close numpy equivalent.